specs/phase0/beacon-chain.md

```diff
@@ -178,6 +178,7 @@ The following values are (non-configurable) constants used throughout the specif
 
 | Name | Value |
 | - | - |
+| `MAX_UINT_64` | `uint64(2**64 - 1)` |
```

Comment:
It's nit-picky, but I think this would be better as `UINT64_MAX`. Where the second `_` is removed and `MAX` is moved to the end. This aligns more with how other libraries do things:

* https://cplusplus.com/reference/climits/
* https://doc.rust-lang.org/std/u64/constant.MAX.html

```suggestion
| `UINT64_MAX` | `uint64(2**64 - 1)` |
```